app/Providers/AppServiceProvider.php

```diff
@@ -18,6 +19,11 @@ public function boot()
         // Fix utf8mb4-related error starting from Laravel 5.4
         Schema::defaultStringLength(191);
 
+        // Enable on delete cascade for sqlite connections
+        if (DB::connection() instanceof \Illuminate\Database\SQLiteConnection) {
```

Comment:
If you could import and use the short name (just like you did `DB`) instead of the full namespaced version I'll be happy to merge this in.